derive_builder/src/options/struct_mode.rs

```diff
@@ -24,7 +27,10 @@ impl OptionsBuilder<StructMode> {
             build_target_vis: ast.vis.clone(),
             builder_name: None,
             builder_vis: None,
+            build_fn_enabled: true,
+            build_fn_name: None,
```

Comment:
Not sure which level of nitpicking is usually applied here. But just want to mention that these should be the first fields to maintain the same order as in the struct def.... :wink: